src/OpenSSL/crypto.py

```diff
@@ -1514,10 +1514,24 @@ def set_flags(self, flags):
 
         :param int flags: The verification flags to set on this store.
             See :class:`X509StoreFlags` for available constants.
-        :return: ``None`` if the verification flags were successfully set.
```

Comment:
heh you removed the wrong one but I see the qualification is everywhere in this class so just re-add this one and let the other be for now